src/components/panel/panel.spec.js

```diff
@@ -117,6 +120,110 @@ describe('$mdPanel', function() {
     expect(parentEl.childElementCount).toEqual(0);
   });
 
+  describe('component logic: ', function() {
+    it('should allow templateUrl to specify content',
+        inject(function($templateCache) {
+          var htmlContent = 'Hello World';
```

Comment:
For each of these tests that use `htmlContent`, could you use a different string? This will protect against the tests accidentally passing because some DOM didn't get cleaned up from a previous test.